docs/docs/recipes/working-with-images.md

```diff
@@ -246,7 +250,7 @@ return (
 )
 ```
 
-f. an image filling a fluid container with a max width (in pixels) and a higher quality (the default value is 50 i.e. 50%)
+F. une image remplissant un récipient de fluide avec une largeur max (en pixels) et une qualité supérieure (la valeur par défaut est 50 soit 50%)
```

Comment:
Please don't change `f.` to `F.` and verify all others points as there are errors

Reply:
solved